### DISCUSSION TITLE
See backlinks to blocks

### DISCUSSION THREAD

**Discussion by pat-richter** [jglev/obsidian-carry-forward#32] [Ideas]
Hi,

thank you so far! Really a helper!

I would like to see where I have used the texts already. When I re-read an excerpt later, I could easily see that (and where) a specific block/line has been used in my thesis or flashcards.

Of course there is an overview in the bottom, but having a small icon (or even number) near the the block would be really helpful. Maybe this is not in the scope of this plugin, but do you think it could be done?

**Comment by jglev** [comment]
Thank you for reaching out! And for your kind words!

> Of course there is an overview in the bottom, but having a small icon (or even number) near the the block would be really helpful. Maybe this is not in the scope of this plugin, but do you think it could be done?

Currently, I don't see this as being in-scope for the Carry-Forward plugin, though I can see how the functionality in Obsidian could be useful! I could see this as being a good scope for a new plugin. Alternatively:

- Running a global Find for a block ID (highlighting the block ID and pressing `Ctrl/Cmd + Shift + F` will show all backlinks to that block, with a count per file.
- The [Influx](https://github.com/jensmtg/influx) plugin provides an additional view of backlinks (albeit more similar to the normal Backlinks view).

This is all to say, I appreciate you reaching out! I don't see this as currently being in the roadmap for Carry Forward, but hope that one of the above alternatives might help for now.

(I'm going to mark this thread as Resolved to reflect it's been addressed, but please feel free to write with any additional thoughts!)

**Reply by pat-richter** [reply]
Thanks for the heads up. It's not the solution I need, but thank you anyway!
